### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by chase-abbott**

### CODE REVIEW

**Comment by chase-abbott** [comment]
Basic requirements of aassignment

File: src/CreatureItem.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        <p> {creature.description} </p>
+        <p> {creature.keyword} </p>
+        <p> Horns: {creature.horns}</p>
+      </li>
+            
+    );
+  }
+}
+
+export default CreatureItem;
```
Comment (dbennin125) [inline src/CreatureItem.js:23]: new line

File: src/CreatureList.css [hunk 2]
```diff
@@ -2,6 +2,6 @@
     list-style: none;
     display: grid;
     grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
-    grid-gap: 10px;
+    grid-gap: 30px;
     justify-items: center;
 }
```
Comment (dbennin125) [inline src/CreatureList.css:7]: new line

File: src/Header.css [hunk 3]
```diff
@@ -1,3 +1,5 @@
 Header {
     background-color: whitesmoke;
+    display: flex;
+    justify-content: center;
 }
```
Comment (dbennin125) [inline src/Header.css:5]: new line

File: src/App.css [hunk 4]
```diff
@@ -1,8 +1,7 @@
...
+  grid-template-rows: auto 1fr auto;
+  grid-gap: 10px
 }
 
-/* TODO: remove **/
-.temp-images {
-  height: 200px;
-}
\ No newline at end of file
+/* TODO: remove **/
```
Comment (dbennin125) [inline src/App.css:7]: new page.

File: src/App.js [hunk 5]
```diff
@@ -13,7 +14,7 @@ class App extends Component {
       <div className="App">
         <Header />
         <main>
-          <CreatureList/>
+          <CreatureList creatures={creatures}/>
```
Comment (dbennin125) [inline src/App.js:17]: sometimes I like to use the word 'prop' to help differentiate things. ex
```
<CreatureList creaturesProp={creatures}/>
```
You would need to reflect the name change in your CreatureList.js 
```
const creatures = this.props.creaturesProp
```
If you understand the concept, disregard but just something I wanted to bring up to help clarify if this doesn't make complete sense yet.

File: src/Footer.css [hunk 6]
```diff
@@ -1,3 +1,4 @@
 .Footer {
-    background-color: rgba(0, 0, 0, .1);
+    background-color: rgba(0, 0, 0, .15);
+    font-size: 1rem;
 }
```
Comment (dbennin125) [inline src/Footer.css:4]: new line

**Pull request merged**